com.endlessm.HatchPreviewer.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Type=Application
+Name=Hatch Previewer
+Icon=com.endlessm.HatchPreviewer
+Exec=/app/bin/run.sh %f
```

Comment:
Is it not supposed to be `/app/bin/hatch-previewer %f`?

Reply:
I'd +1 this

Reply:
Oh, darn. The build stuff for flatpak requires a wrapper script here. But I guess if I call the wrapper script `hatch-previewer`, then the invocations can be the same 😝